RobotEV3/src/main/java/de/fhg/iais/roberta/visitor/validate/Ev3SimValidatorAndCollectorVisitor.java

```diff
@@ -0,0 +1,43 @@
+package de.fhg.iais.roberta.visitor.validate;
+
+import com.google.common.collect.ClassToInstanceMap;
+
+import de.fhg.iais.roberta.bean.IProjectBean;
+import de.fhg.iais.roberta.components.ConfigurationAst;
+import de.fhg.iais.roberta.syntax.action.communication.BluetoothReceiveAction;
+import de.fhg.iais.roberta.syntax.sensor.generic.CompassSensor;
+import de.fhg.iais.roberta.syntax.sensor.generic.HTColorSensor;
+import de.fhg.iais.roberta.syntax.sensor.generic.SoundSensor;
+
+public class Ev3SimValidatorAndCollectorVisitor extends Ev3ValidatorAndCollectorVisitor {
+    public Ev3SimValidatorAndCollectorVisitor(
+        ConfigurationAst robotConfiguration,
+        ClassToInstanceMap<IProjectBean.IBuilder<?>> beanBuilders) {
+        super(robotConfiguration, beanBuilders);
+    }
+
+    @Override
+    public Void visitBluetoothReceiveAction(BluetoothReceiveAction<Void> bluetoothReceiveAction) {
+        addWarningToPhrase(bluetoothReceiveAction, "SIM_BLOCK_NOT_SUPPORTED");
+        return super.visitBluetoothReceiveAction(bluetoothReceiveAction);
```

Comment:
I'm not sure why the call to super is necessary here. All other not supported blocks don't do this.

Reply:
super is needed if there is a sub structure